frontendActions/src/main/resources/actions/ResourceBundle.properties

```diff
@@ -5,9 +5,9 @@ action.GitMachete.CheckoutSelectedBranchAction.text=Checkout Selected Branch
 action.GitMachete.CheckoutSelectedBranchAction.GitMacheteContextMenu.text=_Checkout Branch
 action.GitMachete.CheckoutSelectedBranchAction.description=Checkout this branch
 
-action.GitMachete.RebaseSelectedBranchOntoParentAction.text=Rebase Selected Branch Onto Parent
-action.GitMachete.RebaseSelectedBranchOntoParentAction.GitMacheteContextMenu.text=_Rebase Branch Onto Parent
-action.GitMachete.RebaseSelectedBranchOntoParentAction.description=Rebase this branch onto parent
+action.GitMachete.RebaseSelectedBranchOntoParentAction.text=Checkout and Rebase Selected Branch Onto Parent
```

Comment:
Nit: pls check how other IntelliJ actions choose b/w upper/lower case in words like `and`/`to` in title-cased captions like this one

Reply:
It seems to be the same: words like `and`, `to`, `from`, `the` etc. are in lowercase